### PR TITLE
W-11416060: Add test to the connector for checking prepareArray with XA

### PR DESCRIPTION
**Pull request by giss-ignacio**
Add test to the connector for checking prepareArray and prepare struct with XA transactions initiated in the source

### CODE REVIEW

File: src/test/munit/oracle/arrays-and-structs-xa-source-oracle-test-case.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core"
```
Comment (acurci-at-mulesoft) [inline src/test/munit/oracle/arrays-and-structs-xa-source-oracle-test-case.xml:4]: The namespace ee is never used.

File: src/test/munit/oracle/arrays-and-structs-xa-source-oracle-test-case.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      xmlns:db="http://www.mulesoft.org/schema/mule/db"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:batch="http://www.mulesoft.org/schema/mule/batch"
```
Comment (acurci-at-mulesoft) [inline src/test/munit/oracle/arrays-and-structs-xa-source-oracle-test-case.xml:7]: The namespace batch is never used.

File: src/test/munit/oracle/arrays-and-structs-xa-source-oracle-test-case.xml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                    drop type TEST_STRUCT;
+                </db:sql>
+            </db:execute-script>
+            <error-handler>
+                <on-error-continue logException="false"/>
+            </error-handler>
+        </try>
+    </munit:after-suite>
+
+    <flow name="dbListenerFlow">
```
Comment (acurci-at-mulesoft) [inline src/test/munit/oracle/arrays-and-structs-xa-source-oracle-test-case.xml:90]: @giss-ignacio This flow tests the happy path, could you add another test with a flow with an operation not belonging to the XA transaction, also using var.list and placed between the prepareArray and the store-procedure?

**Review by acurci-at-mulesoft — CHANGES_REQUESTED** [review]

File: src/test/munit/oracle/arrays-and-structs-xa-source-oracle-test-case.xml (hunk 3 above)
Comment (giss-ignacio) [reply src/test/munit/oracle/arrays-and-structs-xa-source-oracle-test-case.xml:90]: @acurci-at-mulesoft According to the new information shared by Cristian Chan, that test cannot be done, right? so we can close this conversation and merge the pr?

**Review by acurci-at-mulesoft — APPROVED** [review]

**Pull request merged**